src/app/todo-list/todo-list.component.html

```diff
@@ -0,0 +1,29 @@
+<h2>TodoList</h2>
+<table>
+  <tr>
+    <th>ID</th>
+    <th>Title</th>
+    <th>Description</th>
+    <th>IsDone</th>
+    <th>Action</th>
+  </tr>
+  <tr *ngFor="let item of items">
+    <td>{{ item.id }}</td>
+    <td>{{ item.title }}</td>
+    <td>{{ item.description }}</td>
+    <td>{{ item.isDone }}</td>
+    <td>
+      <button (click)="OnMarkDone(item.id)" [disabled]="item.isDone">
+        Done
+      </button>
+      <button (click)="onGoToDetail(item.id)">
+        View Detail
+      </button>
+      <button [style.backgroundColor]='"red"' (click)="onDelete(item.id)">
```

Comment:
convenient to use inline style, maybe better to write it into component's css file?